lib/admin/environment/deleteEnv.js

```diff
@@ -0,0 +1,17 @@
+var mbaasRequest = require('../../mbaasRequest/mbaasRequest.js');
+var config = require('../../config/config.js');
+var constants = require('../../config/constants.js');
+
+/**
+ * Checking the status of a DB Migrate in the MBaaS
+ * @param params
```

Comment:
Can you add the parameters to this object that are expected to this JSDOC comment. (domain, environment)